src/main/edit_delete/postProcess.ts

```diff
@@ -0,0 +1,66 @@
+import { Transcription } from 'sharedTypes';
+import Clip from './Clip';
+
+/**
+ * Removes deleted words from edited trancription
+ * @param wordList
+ */
+const removeDeleted = (wordList: Array<any>): void => {
+  let i = 0;
+  while (i < wordList.length) {
```

Comment:
Can do this immutably like so:

`return wordList.filter(word => !word.deleted)`

Bit easier to follow. Also will be O(n) instead of O(n^2)

Reply:
Thanks for the tip! I ended up deleting 'removeDeleted' and just used your one-liner.